app/views/step_by_step_pages/show.html.erb

```diff
@@ -97,6 +98,19 @@
         <% end %>
       </tbody>
     </table>
+
+    <%= render "govuk_publishing_components/components/summary_list", {
+      title: "Sidebar settings",
+      id: "sidebar-settings",
+      edit: {
+        href: step_by_step_page_navigation_rules_path(@step_by_step_page),
+        data_attributes: {
+          gtm: "edit-sidebar-settings"
+        }
+      }
+    } do %>
+      <p class="govuk-body">Guidance around sidebar settings</p>
```

Comment:
This section feels incomplete. Is there guidance to link to or actual guidance that can be displayed here? 

Is there a plan to display the current sidebar settings here?

Reply:
Should the sidebar settings section be hidden if there aren't any navigation rules?

Reply:
Ha, I copied the text exactly as per the design, but didn't stop to read it through properly. You're right, this looks like placeholder text, not sure what's actually supposed to go here.

I'll put a comment on the card when I'm back on my work laptop (or you could, in my absence, to speed things along!)

Tests pass locally & on CI, I wonder if you've run `govuk-docker build` yet? As this PR updates the govuk_publishing_components gem, and adds accessible-autocomplete NPM module. (It's possible `govuk-docker` doesn't account for this but that somehow the Jenkinsfile does - so you may need to run a `yarn install`).

Reply:
I've removed and rebuilt the image and the container, and run `yarn install`, but the tests still fail.

Reply:
Have moved the addition of those sections to a separate PR so that we can get this PR merged.

https://github.com/alphagov/collections-publisher/pull/709

Sounds like we just need to get to the bottom of why the tests are failing on your machine before we proceed 🤔 come find me when you're in the office!

Reply:
Thanks for clarifying the horizontal lines comment. I've added a commit to override the component styles so that the line is only added to the last row item. I'm not sure if this was intended to be the default when Alex raised the [PR adding summary_list to govuk_publishing_components](https://github.com/alphagov/govuk_publishing_components/pull/1061) (and that therefore the style should be changed there) or whether collections-publisher is the only app that will want summary_list without the horizontal lines. It seems like the pragmatic option to perform the override in a commit here, which we can revert in future if the summary_list component is changed.

As for the tests, I've tried deleting my containers and force-deleted the collections-publisher image, and removed my `node_modules` directory. I then rebuilt the collections-publisher image; the tests are still passing.

I notice the [Jenkinsfile has a `beforeTest` step](https://github.com/alphagov/collections-publisher/blob/master/Jenkinsfile#L7-L9) that executes `yarn install`, which is why CI is fine. And [there was a change to govuk-docker](https://github.com/alphagov/govuk-docker/commit/047c905cba5d107b2496e685a8908bf21a8c6523) to install yarn dependancies for collections publisher. Is your `~/govuk/govuk-docker` on `master` and up to date?

Reply:
For the record, we paired on this IRL and couldn't figure out the root cause, but it was fixed on Leena's machine by deleting and re-cloning the repo, and deleting & rebuilding the images & containers.